src/data/services/DiscoveryDataApiService.js

```diff
@@ -97,6 +98,17 @@ class DiscoveryDataApiService {
     const url = `${DiscoveryDataApiService.discoveryBaseUrl}/people/`;
     return apiClient.options(url);
   }
+
+  static autocompletePerson(text, organizationKeys) {
+    let queryString = `?q=${text}&serialize=1`;
+    if (organizationKeys) {
+      organizationKeys.forEach((organizationKey) => {
+        queryString += `&org=${organizationKey}`;
+      });
```

Comment:
You could use ```.reduce``` to be extra fancy here (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce).

Reply:
fancied it up